### ISSUE TITLE
Add a subcommand to print a summary of static catalog

### ISSUE THREAD

**Issue by lossyrob** [stac-utils/stactools#1]
From @cholmes [via Gitter](https://gitter.im/SpatioTemporal-Asset-Catalog/python?at=5f4018d5d4f0f55ebbd0b64b):

> 'catalog summary'. Reports back number of items in the whole catalog, and number of collections.
Perhaps items per collection, but not necessary as I could just point the CLI at a collection directly to get that (since it's also a catalog).

This could be implemented as an `info` command:

```
> stactools info /some/catalog.json
```

a la gdalinfo and friends.

**Comment by m-mohr** [comment]
For extension maturity, it would likely also be interesting if you could get all extensions implemented, divided per specification (catalog, collection, item).

By the way, is PySTAC able to generate the collection summaries field automatically from items?

**Comment by lossyrob** [comment]
> By the way, is PySTAC able to generate the collection summaries field automatically from items?

No, but that's a good feature request. Added https://github.com/stac-utils/pystac/issues/178 to track it, with an additional question for you.

**Comment by cholmes** [comment]
Thanks @lossyrob - I was actually halfway through writing this up, thanks for getting it in. And I'd say it ideally would work on STAC API's as well.

**Comment by lossyrob** [comment]
This is implemented in the [stac info](https://stactools.readthedocs.io/en/latest/cli.html#stac-info) and [stac describe](https://stactools.readthedocs.io/en/latest/cli.html#stac-describe) commands for static STACs. Leaving this open for the inclusion of working with STAC APIs.
